querydsl-apt/src/main/java/com/querydsl/apt/VisitorConfig.java

```diff
@@ -13,8 +13,10 @@
  */
 package com.querydsl.apt;
 
+import com.querydsl.codegen.EntityType;
+
 /**
- * VisitorConfig defines the entity type specific visiting configuration
+ * {@code VisitorConfig} defines the {@link EntityType}-specific visiting configuration
```

Comment:
`{@linkplain }` reads better if you want it to directly flow in the sentence(like it does here).